ass2/css/main.css

```diff
@@ -0,0 +1,388 @@
+body,
+html {
+  font-family: 'Open Sans', sans-serif;
+  margin: 0;
+}
+
+* {
+  -webkit-font-smoothing: antialiased;
+  font-smoothing: antialiased;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+span,
+ul,
+li,
+a,
+button,
+input,
+select,
+textarea,
+table tr td {
+  margin: 0;
+  padding: 0;
+}
+
+ul li {
+  list-style: none;
+}
+
+a {
+  color: #000;
+  cursor: pointer;
+  text-decoration: none;
+}
+input {
+  border: 0;
+  outline: 0;
+}
+
+h1 {font-size: 22px; font-weight: 400;}
+h2 {font-size: 20px; font-weight: 300;}
+h3 {font-size: 18px; font-weight: 300;}
+h4 {font-size: 16px; font-weight: 300;}
+h5 {font-size: 14px; font-weight: 200;}
+h6 {font-size: 12px; font-weight: 200;}
+p,
+ul li,
+a,
+input,
+label,
+select,
+textarea,
+table tr td {
+  font-size: 16px;
+}
+
+button {
+  cursor: pointer;
+}
+
+.clear {
+  clear: both;
+}
+
+.left {
+  float: left;
+}
+
+.right {
+  float: right;
+}
+
+.button {
+  border: 0;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  border-radius: 3px;
+  color: #fff;
+  font-size: 14px;
+  outline: none;
+  padding: 5px 10px;
+  text-transform: uppercase;
+}
+
+.flat-button {
+  border: 0;
+  background-color: transparent;
+  color: #FFA48C;
+  display: block;
+  font-size: 14px;
+  font-weight: 600;
+  text-transform: uppercase;
+}
+
+/*--------navbar--------*/
+
+.navbar-container {
+  height: 60px;
+  position: relative;
+}
+
+.navbar {
+  background-color: #fff;
+  box-shadow: 0 0 3px 0 rgba(0,0,0,0.1);
+  position: fixed;
+  width: 100%;
+  z-index: 100;
+}
+
+.brand-logo {
+  float: left;
+}
+
+.menu {
+  float: left;
+  margin: 15px 50px;
+}
+
+.menu i {
+  font-size: 30px;
+}
+
+.search-container {
+  background-color: #ECEBEB;
+  box-sizing: border-box;
+  border-radius: 3px;
+  float: left;
+  padding: 5px 10px;
+  margin: 15px;
+  width: 40%;
+}
+
+.icon-search:before {
```

Comment:
@priyanka0901 Why you have added icon-search CSS here as it's already available in the icon file.